src/ast.rs

```diff
@@ -83,6 +83,45 @@ impl Value {
     }
 }
 
+// as_xxx().unwrap is safe and None is unreachable
+// when we call it as its correct value
```

Comment:
Safety should be nearby the code that may panics.

Reply:
removed.